.dockerignore

```diff
@@ -1 +1,60 @@
 docker/
+
+# I don't know why the docker folder is being ignored by the docker
+
+# pattern:
+# { term }
+# term:
+# '*' matches any sequence of non-Separator characters
+# '?' matches any single non-Separator character
+# '[' [ '^' ] { character-range } ']'
+# character class (must be non-empty)
+# c matches character c (c != '*', '?', '\\', '[')
+# '\\' c matches character c
+
+# character-range:
+# c matches character c (c != '\\', '-', ']')
+# '\\' c matches character c
+# lo '-' hi matches character c for lo &lt;= c &lt;= hi
+
+# additions:
+# '**' matches any number of directories (including zero)
+# '!' lines starting with ! (exclamation mark) can be used to make exceptions to exclusions
+# '#' lines starting with this character are ignored: use it for comments
+
+
+
+# # Ignore a file or directory in the context root named "modules"
+# modules
+
+# # Ignore any files or directories within the subdirectory named "modules" 
+# # in the context root
+# modules/*
+
+# # Ignore any files or directories in the context root beginning with "modules"
+# modules*
+
+# # Ignore any files or directories one level down from the context root named
+# # "modules"
+# */modules
+
+# # Ignore any files or directories at any level, including the context root, 
+# # named modules
+# **/modules
+
+# # Ignore every file in the entire build context (see next rule for how this 
+# # could be used)
+# *
+
+# # Re-include the file or directory named "src" that may have been previously
+# # excluded. Note that you cannot re-include files in subdirectories that have 
+# # been previously excluded at a higher level
+# !src
+
+
+
+# Set it up so that it removes all git files and unnecessary md and txt tiles  
+git*
+**/*.txt
```

Comment:
I think it would be good to ignore whatever the gitignore does. As 9/10 that will be what we don't need to deploy and copy in a build. I added some updates in this PR #8.